### ISSUE TITLE
Train VGG3 to include higher levels of dropout, data augmentation and batch normalization. 

### ISSUE THREAD

**Issue by Atheer2104** [Atheer2104/dd2424-CNN-classificaiton-project#4]
The target performance of this network but training set-up is ∼ 88%.

Then you should train the network with multiple regularization strategies combined to increase 
performance plus the addition of batch normalization. As you are applying more regularization, 
you might need to train for longer but the use of batch normalization may allow you to use a higher 
learning rate. 

This was also explored in the tutorial so we could just do the same we should get similar results, check the section **Dropout and Data Augmentation and Batch Normalization** in the tutorial for how the final model
